app/discover/procurement/page.tsx

```diff
@@ -0,0 +1,361 @@
+</Text>
+
+<Text>
+The documentation should include, for example, what kind of configurations have been done, how, and where. For new functionalities the names of the new bundles (basically modules of Oskari that do the wanted functionality), their events and requests should be listed. The documentation should be so comprehensive that a person who hasn’t been in the original development team can continue the work.
+</Text>
+
+<Text>
+The documentation can be requested as a separate document or as a Readme file that is included in the GitHub repository of your Oskari instance.
+</Text>
+
+<h3>Support and training</h3>
+
+<Text>
+After your new Oskari service is up and running, you or the people in your organisation might also need support and training. Easiest way to get support might be to join the Oskari Joint development forum and get to know other Oskari main users and organisations, their use cases, problems and wishes. 
+</Text>
+
+<Text>
+You can also procure support and training for Oskari and the software related to it from commercial partners. In some other procurement cases the experience with Oskari is not necessarily required from the vendor, but with support and training it is highly required.
+</Text>
+
+<h3>When can you use your own organisation&apos;s IT capabilities?</h3>
+
+<Text>
+If you have an in-house IT department, there are certain Oskari processes that can be done by them. These are also some of the easier things to do with Oskari that don’t require Oskari specific knowledge or expertise. The Oskari documentation is also being improved so that as many people as possible could use Oskari and join in the development.
+</Text>
+
+<Text>
+<ul>
+Database-related work:
+<li>- Updating and maintenance of databases</li>
+<li>- Updating and maintenance of database connections</li>
+</ul>
+</Text>
+
+<Text>
+<ul>
+HTML, CSS, SCSS experience:
+<li>- Competence related to Oskari&apos;s front-end and its configurations</li>
+</ul>
+</Text>
+
+<Text>
+<ul>
+??? experience:
+<li>- Updating Oskari (step-by-step instructions in <Link href='https://github.com/oskariorg/oskari-server/blob/master/MigrationGuide.md'>the migration guide</Link>)</li>
+</ul>
+</Text>
+
+<h2>Closing thoughts</h2>
+<Text>
+Should you have any questions about your wishes, use cases or problems with Oskari, please contact <Link href='/community'>the project coordinators</Link>. Ask for help and tell your developers to contact us if they are facing problems. Report bugs by making issues on <a href='https://github.com/oskariorg/oskari-documentation/issues'>Oskari&apos;s GitHub repository</a> and create pull requests that include your proposals for new, better solutions both in Oskari software and its documentation. As a procurer of Oskari-related projects, try to make sure that the new admin, developer or end-user of your Oskari service has an easy start with Oskari. That is our aim, too.
```

Comment:
From earlier note about "NLS FI Oskari team", should we use project coordinaters everywhere here as the main contact point if not a public one (github, gitter, mailing list)?

Reply:
Will leave this open until either a Contact us page is done (by yours truly) or if the idea is scrapped, revise the text